### PR TITLE
Assignment-2: Event handling & DOM

### PR DESCRIPTION
**Pull request by vaibhav-webonise**
**Problem 1:** Write a JavaScript code that starts to execute on a button click and will open a particular url in multiple new tabs (say 5) and close them after a certain interval recursively. The script should stop its execution onclick of another button on the same page.

**Problem 2:** Create a simple form that will accept name and email of a user and onsubmit, stores the details with the current timestamp. User can add multiple name and email sets. There should  be a button “SHOW USERS” which will list all the name and email sets with their timestamp in the document. Form should be validated before submission else it should show an error. (Expected session handling for this)

### CODE REVIEW

File: EventHandlingDemo.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (swapnil-webonise) [inline EventHandlingDemo.html]: Again same thing repeated here, Indentation improper

File: ValidateUser.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <label for="emailOfUser">Email Id :</label><input type="email" id="emailOfUser">
+    <button type="button" onclick="logIn()">Submit</button>
+    <button type="button" onclick="getData()">Show data</button>
+</form>
+
+<script>
+    sessionStorage = [];
+
+    function logIn() {
+        let userName = document.getElementById("nameOfUser").value;
```
Comment (swapnil-webonise) [inline ValidateUser.html]: Use single quotes

File: ValidateUser.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        }
+    }
+
+    function getData() {
+        for (let i = 0; i < sessionStorage.length; i++) ;
+        {
+            let currentTime = JSON.parse(sessionStorage.getItem("currentTime"));
+            let userEmail = JSON.parse(sessionStorage.getItem("userEmail"));
+            let userName = JSON.parse(sessionStorage.getItem("userName"));
+            document.write("<b>User Name :</b> " + userName);
```
Comment (swapnil-webonise) [inline ValidateUser.html]: Use template string for concatenation

**Review by swapnil-webonise — CHANGES_REQUESTED** [review]

File: EventHandlingDemo.html (hunk 1 above)
Comment (vaibhav-webonise) [reply EventHandlingDemo.html]: that one line after body and button tag or anything else in the file?

Comment (swapnil-webonise) [reply EventHandlingDemo.html]: everything should be indented with 2 space tab

**Review by swapnil-webonise — APPROVED** [review]

**Pull request merged**